tracer/src/Datadog.Trace/Telemetry/DTOs/ConfigTelemetryData.cs

```diff
@@ -27,12 +29,16 @@ internal static class ConfigTelemetryData
         // We intentionally are using specific values here, not OR_GREATER_THAN
 #if NET6_0
         public const string ManagedTracerTfmValue = "net6.0";
+        internal const TargetFrameworks TargetFramework = TargetFrameworks.NET6_0;
 #elif NETCOREAPP3_1
         public const string ManagedTracerTfmValue = "netcoreapp3.1";
+        internal const TargetFrameworks TargetFramework = TargetFrameworks.NETCOREAPP3_1;
 #elif NETSTANDARD2_0
         public const string ManagedTracerTfmValue = "netstandard2.0";
+        internal const TargetFrameworks TargetFramework = TargetFrameworks.NETSTANDARD2_0;
 #elif NETFRAMEWORK
         public const string ManagedTracerTfmValue = "net461";
+        internal const TargetFrameworks TargetFramework = TargetFrameworks.NET461;
```

Comment:
nit: Let's just make them public for consistency? 😄 
```suggestion
        public const string ManagedTracerTfmValue = "net6.0";
        public const TargetFrameworks TargetFramework = TargetFrameworks.NET6_0;
#elif NETCOREAPP3_1
        public const string ManagedTracerTfmValue = "netcoreapp3.1";
        public const TargetFrameworks TargetFramework = TargetFrameworks.NETCOREAPP3_1;
#elif NETSTANDARD2_0
        public const string ManagedTracerTfmValue = "netstandard2.0";
        public const TargetFrameworks TargetFramework = TargetFrameworks.NETSTANDARD2_0;
#elif NETFRAMEWORK
        public const string ManagedTracerTfmValue = "net461";
        public const TargetFrameworks TargetFramework = TargetFrameworks.NET461;
```